src/core/session.c

```diff
@@ -28,6 +28,12 @@ static int method_close(sd_bus_message *msg, void *data,
 
 	sd_bus_message_unref(reply);
 
+	char *exec_after = sess->screencast_instance->ctx->state->config->screencast_conf.exec_after;
+	if (exec_after) {
+		logprint(INFO, "xdpw: executing %s after screencast", exec_after);
+		system(exec_after);
```

Comment:
This blocks the whole xdpw process while the command is being executed. If think it'd be preferable to fork_exec instead.

https://github.com/emersion/xdg-desktop-portal-wlr/pull/59 has some code for this (although the pipe communication is not needed here).